content/docs/contributing/to-docs/blog-posts.md

```diff
@@ -0,0 +1,106 @@
+    date: 2021-07-06
+    slug: welcome-post
+    author: Doug Erickson, Amazon Web Services
+    blog_img: "/images/blog/announcement_thumbnail.jpg"
+    full_img: "/images/blog/atom_showcase.png"
+    ---
+    ```
+
+    Note that the blog is automated and that the `date` field determines when the blog will first become visible if your pull request is accepted and merged.
+6. Commit your changes to your branch. **Ensure your commit(s) are DCO signed!***
+
+7. Push your branch to your fork (`origin`). Go to your fork on GitHub and create a pull request from your fork/branch to o3de/o3de.org/main (`upstream`).
+
+8. Go to your pull request at https://github.com/o3de/o3de.org/pulls and ensure that all checks have passed. If they haven't, fix up your pull request and resubmit it.
+
+9. Go into the O3DE Discord server and share your pull request for review. (Alternatively, mail the sig-docs-community mailing list with a link.) Someone will review it and provide feedback or an okay. A maintainer will merge it when at least one reviewer approves it.
+
+{{< note >}}
+The O3DE Community reserves the right to not publish any blog post that does not meet our standards. We also reserve the right to remove any currently published blog post that might conflict with the agreed-upon O3DE strategy or delivery.
+{{< /note >}}
+
+## What makes a good O3DE blog post?
+
+Good question! That's really up to the community alongside our established Code of Conduct and Community Tenets. Not sure? Join the O3DE Discord or one of the SIGs, and vet your idea with our members.
+
+Here's some basic Do's and Don'ts to help you scope your effort and avoid an obvious rejection.
+
+**Do:**
+
+* Discuss O3DE! (Duh.) Go deep! Highly technical blog posts, in particular, establish community credibility and provide a resource for people looking for deeper knowledge beyond the tech docs.
+
+* Discuss algorithms, techniques, technologies, and tools that apply directly to O3DE work. Again, not only is your knowledge and experience useful to capture as a published topic, but it helps the larger industry assess the caliber of the people building O3DE.
+
+* O3DE strategy, **if** you are actively participating in a SIG around feature or product strategy, **and** have the approval of a SIG.
+
+* Interviews with, or profiles of, important and established community members. People put a human face on a technical product, and the world reliably likes to hear humans say they have to say. In this case, it should be professional observations, knowledge, and commentary&mdash;and shuld be specifically related to O3DE. Personal opinions, as ever, should be largely technical.
+
+* Write with strong grammar and an eye for easy readability. Rambling monologues may get a gentle request for editorial review and a number of suggestions to "tighten it up". The reader's time is precious, so compressing your thoughts into short, information-dense sentences with regular paragraph breaks is just respectful.
```

Comment:
```suggestion
* Write with strong grammar and an eye for easy readability. Rambling monologues may get a gentle request for editorial review and a number of suggestions to "tighten it up." The reader's time is precious, so compressing your thoughts into short, information-dense sentences with regular paragraph breaks is just respectful.
```

Reply:
Getting Docs to take a look on this to see if it should be changed or not.